src/Parser/MultipleParser.php

```diff
@@ -22,7 +22,10 @@ public function __construct(Parser ...$parsers)
         $this->parsers = $parsers;
     }
 
-    /** @psalm-return list<non-empty-string> */
+    /**
+     * @return array
+     * @psalm-return list<never>
```

Comment:
What is never?

Reply:
It complies with the [never](https://www.php.net/manual/en/language.types.never.php) type in PHP.

Reply:
TIL 🙌🏻